### ISSUE TITLE
[Non-text content and media] Non-text content doesn't have a text alternative. (10.01.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59461]
#### VFS team name
MHV

#### VFS product name
My HealtheVet

#### VFS feature name
Secure Messaging

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Non-text content doesn't have a text alternative.

### Issue Details
When viewing a list of messages (eg. [/inbox/](https://staging.va.gov/my-health/secure-messages/inbox)), unread messages are indicated by bold text and a circle icon to the left of the message. Bold text is not typically communicated through a screen reader, and the icon doesn't have any alternative text, or any alternative way of communicating that an attachment is present (eg. extra `aria-label` on another element). For screen reader users, there's no way of telling which messages are unread and which have been read.

### Link, screenshot or steps to recreate
```
<div class="thread-list-item vads-l-row vads-u-padding-y--1p5 vads-u-border-bottom--1px vads-u-border-color--gray-light" data-testid="thread-list-item">
   <div class="unread-column vads-l-col">
      <i aria-hidden="true" class="unread-icon vads-u-margin-right--1 vads-u-color--primary-darker fas fa-solid fa-circle" data-testid="thread-list-unread-icon"></i>
   </div>
   <div class="vads-l-col vads-u-margin-left--1">
      <div class="vads-u-margin-y--0p5 vads-u-font-weight--bold">
          <span><span class="thread-list-draft">(Draft)</span> - </span> <span>ISLAM, MOHAMMAD  RAFIQ (Team: ***MEDICATION_AWARENESS_100% @ MOH_DAYT29)</span> <span class="message-count">(3 messages)</span>
      </div>
      <a class="message-subject-link vads-u-margin-y--0p5" href="/my-health/secure-messages/thread/2542672">General: FREEMAN: "RETRY" PUSH NOTIFICATION</a>
      <p class="received-date vads-u-margin-y--0p5"><span>November 23, 2022 at 1:58 p.m. CST</span></p>
   </div>
</div>
```

Note that the unread message icon `<i>` has no text equivalent indicating that the message is unread.

### VA.gov Experience Standard
[Category Number 10, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.1.1_A

### Platform Recommendation
There are several ways you could add some kind of text alternative, but I think what I'd recommend here is inserting a `<span class="sr-only">Unread message</span>` after the `<i>` icon. I would also recommend associating that text alternative with the message subject link using `aria-describedby`, so that a screen reader user who is tabbing through all the message links hears that the message is unread without having to explore the context around the message.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by bryan-riley-va** [comment]
JIRA ticket created: https://jira.devops.va.gov/browse/MHV-45987

**Comment by AngelaFowler82** [comment]
I can tell messages are unread by arrowing down the page and tabbing through the links. Closing this as resolved.
